src/components/Home.jsx

```diff
@@ -23,14 +23,37 @@ const Home = () => {
     history.push('/list-view');
   };
 
+  const compareToken = (e) => {
+    e.preventDefault();
+    console.log(tokenName);
+
+    db.collection('tokens')
+      .where('tokenArray', 'array-contains', tokenName)
+      .get()
```

Comment:
Super great work, I have learned a lot from what have guys done. One thing I would like to mention is in your compare token function. It looks like you pulling some asynchronous data, This is causing some warning because you haven't applied something like effect hooks to pull those data.  
![error](https://user-images.githubusercontent.com/43877689/127436901-0ede2188-86e2-4c28-b1c9-7e00c295e00e.PNG)

Reply:
Thanks for flagging the error in the console! I'm actually not getting this on my side so it's really useful. I'll have a look and see if I can replicate it locally and fix it. Thank you! ☺️

Reply:
Also got this, will look out for it during the review.

Reply:
Yep, I see the same console message on my end too. This could totally have to do something with useEffect hook.